src/IO/H5/VolumeData.cpp

```diff
@@ -306,6 +306,29 @@ std::vector<std::vector<size_t>> VolumeData::get_extents(
   return individual_extents;
 }
 
+std::pair<size_t, size_t> offset_and_length_for_grid(
+    const std::string& grid_name,
+    const std::vector<std::string>& all_grid_names,
+    const std::vector<std::vector<size_t>>& all_extents) noexcept {
+  auto found_grid_name =
+      std::find(all_grid_names.begin(), all_grid_names.end(), grid_name);
```

Comment:
If you include `Utilities/Algorithm.hpp` you can do `alg::find(all_grid_names, grid_name)`, which is easier to read :)